package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "counter-invariant-tests-tests",
+  "version": "1.0.0",
+  "description": "Run unit tests on this project.",
+  "private": true,
+  "scripts": {
+    "test": "vitest run",
+    "test:report": "vitest run -- --coverage --costs",
+    "test:watch": "chokidar \"unit-tests/**/*.ts\" \"contracts/**/*.clar\" -c \"npm t\""
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@hirosystems/clarinet-sdk": "^1.0.1",
+    "@stacks/transactions": "^6.9.0",
+    "chokidar-cli": "^3.0.0",
+    "fast-check": "^3.13.1",
+    "typescript": "^5.2.2",
+    "vite": "^4.4.9",
+    "vitest": "^0.34.4",
+    "vitest-environment-clarinet": "^1.0.2"
```

Comment:
Where can I find more info about this package?

Reply:
The readme is really short, but so is the index.js
https://github.com/hirosystems/vitest-environment-clarinet/

It sets up some global variables and initializes the simnet